runtime/common/logging_xwalk.cc

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2015 The Chromium Authors. All rights reserved.
```

Comment:
I bet those two files copied from src/chrome/logging_chrome.[cc|h], please leave 2012 chromium copyright, and add Intel copyright in.